src/components/activities/ActivityCard.jsx

```diff
@@ -79,6 +79,15 @@ class ActivityCard extends Component {
     wordCount: 50,
   };
 
+  constructor(props) {
```

Comment:
Static class properties/methods should be defined before the constructor. Eslint should have caught this

Reply:
@Elbertbiggs360, ☝️ This has occured in multiple components, please revert these changes